index.js

```diff
@@ -33,9 +39,9 @@ module.exports = function(glob, options) {
 
   // make glob absolute
   if (rootDir && glob.charAt(0) === '/') {
-    glob = path.join(rootDir, glob);
-  } else if (!isAbsolute(glob)) {
-    glob = path.resolve(cwd, glob);
+    glob = join(rootDir, glob);
+  } else if (process.platform === 'win32' || !isAbsolute(glob)) {
```

Comment:
Scratch those 2 things.  This is solved by removing the `process.platform === 'win32'` check.  The `!isAbsolute` call is enough to determine if this branch is needed.